src/marqo/tensor_search/models/add_docs_objects.py

```diff
@@ -29,7 +29,7 @@ class Config:
     imageDownloadHeaders: dict = Field(default_factory=dict)
     modelAuth: Optional[ModelAuth] = None
     mappings: Optional[dict] = None
-    documents: Union[Sequence[Union[dict, Any]], np.ndarray]
+    documents: Union[Sequence[Union[dict, Any]], list]
```

Comment:
Hello @gabauer. I think we can just use `Sequence[dict[str, Any]]` type for documents and docs parameter below.